src/test/java/com/wikia/webdriver/elements/mercury/pages/InfoboxBuilderPage.java

```diff
@@ -1,16 +1,16 @@
-package com.wikia.webdriver.elements.oasis.pages;
+package com.wikia.webdriver.elements.mercury.pages;
 
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
 import com.wikia.webdriver.common.core.Assertion;
```

Comment:
We already tried to separate f.e. modal as separate component, but then we realized that with current implementation, we are unable to reuse it.

I agree with splitting that class, although I would like to do this as separate task. The reason for that is this feature needs that tests asap.